src/pkg/cli/client/byoc/clouds/aws.go

```diff
@@ -209,8 +209,19 @@ func (b *ByocAws) Deploy(ctx context.Context, req *defangv1.DeployRequest) (*def
 	}
 	b.cdTasks[etag] = taskArn
 
+	deployed, err := b.GetServices(ctx)
```

Comment:
This sounds tricky. Could be a race: either the CD hasn't upload the blob yet, or we get a blob from a previous deployment

Reply:
Yes, agreed, actually this is no longer needed, I wanted to display the ALB url here so the user can setup DNS, but I decided to remove it last minute as it would be a conflicting instruction that is given from the `cert gen` command